frontend/src/App.tsx

```diff
@@ -1,36 +1,93 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
+import styled from 'styled-components';
 import logo from './logo.svg';
-import './App.css';
- 
-class App extends Component {
- 
-    state:any = {};
- 
-    componentDidMount() {
-        setInterval(this.hello, 250);
+import LandingPage from './views/Landing';
+import fetchHello from './api/hello';
+
+const StyledApp = styled.div`
```

Comment:
Not a suggested change, just a note: If @alankbi or @helenli8181 haven't done so yet, if either of you are using Visual Studio Code, I recommend adding the [vscode-styled-components](https://marketplace.visualstudio.com/items?itemName=jpoissonnier.vscode-styled-components) extension to help with syntax highlighting -- this makes the code much more readable.

Reply:
Similarly, there's a styled components plugin for IntelliJ too. And while we're on the topic of IDE settings, I added ESLint in this PR but I think you'll still need to enable linting in VSCode settings.